site/cypress/e2e/pdfs.spec.js

```diff
@@ -0,0 +1,105 @@
+import {getCurrentSemester} from '../support/utils.js';
+
+function switch_settings(setting, pdf_type) {
+    cy.visit(`/courses/${getCurrentSemester()}/sample`);
```

Comment:
The `/courses/<semester>` is unnecessary here.  You can just use `cy.visit(['sample'])`, like the code [here](https://github.com/Submitty/Submitty/blob/36c58b64c779c4141bf4be995f2bf79dae2e4b03/site/cypress/e2e/late_days_allowed.spec.js#L4).  Apply throughout the rest of this PR.

This particular line is also completely unnecessary because you navigate to a page, and then promptly navigate away to log in.